src/runtime/virtcontainers/mock_agent.go

```diff
@@ -176,12 +176,12 @@ func (n *mockAgent) resumeContainer(ctx context.Context, sandbox *Sandbox, c Con
 	return nil
 }
 
-// configHypervisor is the Noop agent hypervisor configuration implementation. It does nothing.
-func (n *mockAgent) configure(ctx context.Context, h hypervisor, id, sharePath string, config interface{}) error {
+// configure is the Noop agent configuration implementation. It does nothing.
```

Comment:
nit: `config` instead of `configure`

Reply:
Hi @fidencio, `configure` is the function name but not the parameter name, though the function is not exported, according to Golang's style, the first word should be the function name.